dashboard/src/util/Loading.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { Dimmer, Loader, Segment } from 'semantic-ui-react';
+
+class Loading extends Component {
+  render() {
```

Comment:
Again, if this doesn't need state or anything special, just use a function component here. Also helps increasing the visibility of what props it takes.